langkit/templates/memoization_ada.mako

```diff
@@ -302,19 +304,19 @@ end Find_Memoized_Value;
 ------------------------
 
 procedure Add_Memoized_Value
-  (Unit       : Internal_Unit;
-   Handle     : in out Memoization_Handle;
-   Value      : Mmz_Value) is
+  (Unit   : Internal_Unit;
+   Handle : in out Memoization_Handle;
+   Value  : Mmz_Value;
+   Stored : out Boolean) is
 begin
    --  If Handle was created using a memoization map that has been since then
    --  reset, do nothing: the result can be partly stale due to the event that
    --  triggered the memoization tables reset.
 
-   if Handle.Cache_Version < Unit.Cache_Version then
-      return;
+   Stored := Unit.Cache_Version <= Handle.Cache_Version;
```

Comment:
This is not the inverse of the original condition (`<=` instead of `>=`). Not sure if it's expected or not, just mentioning in case it's a typo.

Reply:
FTR, the operands are switched, so this *is* the negation of the original condition. Sorry for the confusion, and thanks for the review!. ;-)